components/constants/brave_switches.h

```diff
@@ -27,6 +27,8 @@ extern const char kDarkMode[];
 extern const char kDisableMachineId[];
 
 extern const char kDisableEncryptionWin[];
+  
+extern const char kKeepOldHistory[];
```

Comment:
this switch should be put into `history` component. this whole file shouldn't be used for new switches.